android/titanium/build.gradle

```diff
@@ -121,6 +125,24 @@ android {
 	}
 }
 
+// Install cmake during project configuration.
+def cmakeDir = new File(sdkDir+'/cmake/'+android.externalNativeBuild.cmake.version)
```

Comment:
I've reverted updating cmake, however maybe we should keep this check here so we can easily update cmake in the future?

Reply:
Yeah, I think this check is fine. Thanks.